start/src/main/java/io/openliberty/guides/config/CustomConfigSource.java

```diff
@@ -25,86 +25,5 @@
 import org.eclipse.microprofile.config.spi.ConfigSource;
 
 // tag::customConfig[]
-public class CustomConfigSource implements ConfigSource {
```

Comment:
Why are we removing the custom config source?

Reply:
This file is in the start folder, which should be empty, and the guide will ask the user to add the custom config source class.

Reply:
Aa, I see.  Makes sense then.